bin/stats-resources

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+php /usr/src/code/app/cli.php stats-resources $@
```

Comment:
this should be task-stats-resources right?

Reply:
the filename atleast

Reply:
Yes, will take care of it in separate PR as none of the tasks follow this convention yet.